mobile/lib/services/location_service.dart

```diff
@@ -174,7 +174,7 @@ class LocationService {
       position.latitude,
       position.longitude,
     );
-    airQualityReadings = airQualityReadings.sortByDistanceToReferenceSite();
+    airQualityReadings = airQualityReadings..sortByDistanceToReferenceSite();
 
```

Comment:
please enlighten me on the double dots operator here? @NoahNsimbe

Reply:
This is cascading notation. It allows you to make a sequence of operations on the same object 
https://dart.dev/language/operators#cascade-notation

Reply:
cool